.github/CONTRIBUTING.md

```diff
@@ -75,60 +76,12 @@ By default we provide various automated checks you can leverage to test your cha
 - Click the "Checks" tab at the top of the pull request. **OR**
 - Click the "Details" link next to the list of checks that shows up at the bottom of the pull request. If the tests passed, this list will be hidden, so you will first need to click the "Show all checks" link.
 
+Currently, some our automated checks are internal to Splunk and their details cannot be publicly viewed. The details for the following checks *can be* publicly accessed. 
```

Comment:
```suggestion
Currently, some our automated checks are internal to Splunk and their details cannot be publicly viewed. The details for the following checks *can* be publicly accessed. 
```

your call on what you intended with the emphasis here, but I think you meant to just emphasize `can`?

Reply:
will update